client/src/components/home/WishlistIcon.js

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart } from '@fortawesome/free-solid-svg-icons';
+import { useSelector } from "react-redux";
+import { selectUser } from "../../features/auth/auth.selectors";
+import axios from 'axios';
+
+const removeSeriesFromWishlist = async (series, user) => {
```

Comment:
Move both these methods to series service file (in `services` package).
import series service here and call these methods from the service.
Same here, use config file for `localhost:8080`
Might consider merging these methods to one and simply send a parameter for add or remove